src/libraries/ByteHasher.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+library ByteHasher {
+    /// @dev Creates a keccak256 hash of a bytestring.
+    /// @param value The bytestring to hash
+    /// @return The hash of the specified value
+    function hash(bytes memory value) internal pure returns (uint256) {
```

Comment:
The `>> 8` makes this function more than just a simple hash function, it also ensures the result is reduced field element (i.e. less than the prime modulus). A name like `hashToField` would be more approrpriate. It's also worth mentioning it in the docs.